README-docker-compose.md

```diff
@@ -211,8 +211,8 @@ $ docker-compose run --rm --service-ports web
   - Local host name: `192.168.168.167`
   - Port: `11000`
   - Path mappings: (It is recommended to use absolute path. `~/` may not work.)
-    - `/Users/<your username>/Projects/cos/osf : /code`
-    - (Optional) `/Users/<your username>/.virtualenvs/osf/lib/python2.7/site-packages : /usr/local/lib/python2.7/site-packages`
```

Comment:
* Spaces are not allowed before or after the `:`
* Locally verified that the updated mapping works.